data/summary/types/bgm_test.go

```diff
@@ -322,53 +316,69 @@ var _ = Describe("BGM Summary", func() {
 				Expect(err).ToNot(HaveOccurred())
 				err = types.AddData(&userBGMSummary.Stats.Buckets, dataSetBGMDataFive)
 				Expect(err).ToNot(HaveOccurred())
+				err = types.AddData(&userBGMSummary.Stats.Buckets, dataSetBGMDataSix)
+				Expect(err).ToNot(HaveOccurred())
 
-				Expect(len(userBGMSummary.Stats.Buckets)).To(Equal(5))
+				Expect(len(userBGMSummary.Stats.Buckets)).To(Equal(6))
 
 				By("check record counters for insurance")
 				for i := len(userBGMSummary.Stats.Buckets) - 1; i >= 0; i-- {
 					f := fmt.Sprintf("hour %d", i+1)
 					By(f)
-					Expect(userBGMSummary.Stats.Buckets[i].Data.TotalRecords).To(Equal(5))
+					Expect(userBGMSummary.Stats.Buckets[i].Data.TotalRecords).To(Equal(6))
 
-					lastRecordTime = datumTime.Add(-time.Hour*time.Duration(len(userBGMSummary.Stats.Buckets)-i-1) - time.Minute*12)
-					Expect(userBGMSummary.Stats.Buckets[i].LastRecordTime).To(Equal(lastRecordTime))
+					//lastRecordTime = datumTime.Add(-time.Hour*time.Duration(len(userBGMSummary.Stats.Buckets)-i-1) - time.Minute*10)
+					// TODO
```

Comment:
Is this still to be done?

If not please remove, if so, please... do?

If you end up back in here, consider the `var lastRecordTime` on line 292 as well.

Reply:
Uncommented, was a fully working check I only temporarily needed to bypass, good catch.